doc/source/parameters/sharp-immersed-boundary-solver/sharp-immersed-boundary-solver.rst

```diff
@@ -255,6 +253,8 @@ The following parameter and subsection are all inside the subsection ``particle
 
 * The ``integrate motion`` parameter controls if the dynamics equations of the particles are calculated. If this parameter is set to false, the particles position, velocity, and angular velocity are defined directly by the functions. If ``integrate motion=true`` the position and the velocity will be defined by the integration of the particle dynamic.
 
+* The ``mesh based precalculations`` parameter controls if the mesh-based precalculations are applied. These precalculations are critical for good performance in medium to high detailed RBFs (and its composites), but can introduce deformations. These deformations appear when some RBF nodes are located outside of the background mesh.
```

Comment:
So should this be set to false for all RBF cases? In this case, I would suggest it there

Reply:
Not always, only for cases where some RBF nodes are outside of the background grid and where perfect shape representation is needed. For example, in MMS cases where we use symmetry and the RBF shape is cut by the grid boundary, we must deactivate mesh precalculations. However, for large simulations of complex shapes we must leave ``mesh-based precalculations=true``, otherwise all cells would have to loop on all RBF nodes, making it too costly. I plan to fix this issue soon and assign each outside node to its closest boundary cell so we can keep mesh-based precalculations active.